### PR TITLE
Accessbility 8 2023

### PR DESCRIPTION
**Pull request by chensation**
Fixed accessibility bugs from 8/2023 pass:
Sev 1:
[24742265](https://msazure.visualstudio.com/One/_queries/edit/24742265/?triage=true) info icon on apptype viewer not keyboard accessible. 

Sev2:
query: https://msazure.visualstudio.com/One/_queries/query/?tempQueryId=67a6f913-dfe5-4b5f-840e-4cf164fb24ec

24720524 Arrow button to close nav menu did not have compliant text, changed it to a title property.

24740449 Luminosity contrast on toggles too low, increased contrast

24742303 Screen Reader didn't read out the options for the drop-down menu on the powershell command parameters properly, changed it.

24744004 Correlated button on the event tab had inappropriate role attribute, change it to a toggle to avoid complications. Changed all similar implementations (create service modal) to proper radio buttons

24753246 Info icon for event list on the event tab not keyboard accessible, fixed it. 

24754870 After setting viewport 320*256 pixels, 'Export' & 'Cancel' control on some modals not viewable, changed all modal's display logic to account for screen size changes.

### CODE REVIEW

File: src/SfxWeb/src/app/modules/apptypes-viewer/apptype-viewer/apptype-viewer.component.html [hunk 1]
```diff
@@ -1,39 +1,37 @@
-<div class="tile-container" data-cy="tiles">
-  <app-dashboard-text-tile [barClass]="'badge-upgrade-active'" [count]="usage.activeAppTypes.length" *ngIf="usage"
```
Comment (jeffj6123) [inline src/SfxWeb/src/app/modules/apptypes-viewer/apptype-viewer/apptype-viewer.component.html:2]: did the tiles get fully removed or just moved?

**Review by jeffj6123 — CHANGES_REQUESTED** [review]

File: src/SfxWeb/src/app/views/application/create-service/create-service.component.scss [hunk 2]
```diff
@@ -17,26 +17,8 @@ dl {
     }
 }
 
-.solid-grey {
```
[the remark this answers is not in the document]
Comment (chensation) [reply src/SfxWeb/src/app/views/application/create-service/create-service.component.scss:20]: yes, these styling were used for the previous toggle style button, now the radio buttons don't need them.

File: src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.html [hunk 3]
```diff
@@ -24,11 +24,13 @@ <h2 collapse-header>
...
-  <div style="height: 20px;">
-    <a class="link" style="float: right;"> Why is data missing?
-      <span class="mif-info"
-        title="This timeline may present incomplete information and not fully represent the cluster for this date range. Events may also be missing due to data loss and lead to incomplete states. E.g.: A cluster upgrade starting before or ending after this time range will not be shown."></span>
-    </a>
+  <div style="height: 20px;"> 
+    <span style="float: right;"
+    > Why is data missing?
+      <span class="mif-info"  ngbTooltip="This timeline may present incomplete information and not fully represent the cluster for this date range. Events may also be missing due to data loss and lead to incomplete states. E.g.: A cluster upgrade starting before or ending after this time range will not be shown."
+      tooltipClass="styled-tooltip" aria-label="Tooltip - Why is data missing? This timeline may present incomplete information and not fully represent the cluster for this date range. Events may also be missing due to data loss and lead to incomplete states. E.g.: A cluster upgrade starting before or ending after this time range will not be shown."
```
[the remark this answers is not in the document]
Comment (chensation) [reply src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.html:31]: removed, I suppose it is not an actual tooltip

File: src/SfxWeb/src/app/modules/apptypes-viewer/apptype-viewer/apptype-viewer.component.html (hunk 1 above)
Comment (chensation) [reply src/SfxWeb/src/app/modules/apptypes-viewer/apptype-viewer/apptype-viewer.component.html:2]: it has been moved to apptypes.component.html

**Review by jeffj6123 — APPROVED** [review]

**Pull request merged**